src/main/java/com/github/tomakehurst/wiremock/http/ProxyResponseRenderer.java

```diff
@@ -105,11 +105,21 @@ public Response render(ServeEvent serveEvent) {
                     )
                     .chunkedDribbleDelay(responseDefinition.getChunkedDribbleDelay())
                     .build();
-		} catch (IOException e) {
-			return throwUnchecked(e, null);
-		}
+        } catch (SSLException e) {
+            return proxyResponseError("SSL", httpRequest, e);
+        } catch (IOException e) {
+            return proxyResponseError("Network", httpRequest, e);
+        }
 	}
 
+
+    private Response proxyResponseError(String type, HttpUriRequest request, Exception e) {
+        return response()
```

Comment:
How about we log the error here too, so that the full stack trace is available if you've got access

Reply:
Pushed a change; I've logged it as info, as it's normal behaviour to make a request to an endpoint that doesn't exist / is down / has no valid cert